### ISSUE TITLE
What does yolov3-tiny.yaml mean?

### ISSUE THREAD

**Issue by jaskiratsingh2000** [ultralytics/yolov5#3421]
Hey, I just wanna ask that what does yolov3-tiny.yaml mean? https://github.com/ultralytics/yolov5/blob/develop/models/hub/yolov3-tiny.yaml
Is there a way that I could measure the profiling for yolo-tiny version what can be done easily within yolov5 within models/yolo.py ?

Please let me know. Thanks! @glenn-jocher

**Comment by glenn-jocher** [comment]
@jaskiratsingh2000 yolov3-tiny.yaml is the yaml used to create YOLOv3-tiny models. You can use it to start training from scratch with this repo. For pretrained YOLOv3-tiny models see https://github.com/ultralytics/yolov3

**Comment by jaskiratsingh2000** [comment]
@glenn-jocher What I mean to say that how can I produce the profiling like the below for YOLOv3-tiny?

```
YOLOv5 🚀 v5.0-100-g4a8d238 torch 1.7.0a0+e85d494 CPU


                 from  n    params  module                                  arguments                     
  0                -1  1      3520  models.common.Focus                     [3, 32, 3]                    
  1                -1  1     18560  models.common.Conv                      [32, 64, 3, 2]                
  2                -1  1     18816  models.common.C3                        [64, 64, 1]                   
  3                -1  1     73984  models.common.Conv                      [64, 128, 3, 2]               
  4                -1  1    156928  models.common.C3                        [128, 128, 3]                 
  5                -1  1    295424  models.common.Conv                      [128, 256, 3, 2]              
  6                -1  1    625152  models.common.C3                        [256, 256, 3]                 
  7                -1  1   1180672  models.common.Conv                      [256, 512, 3, 2]              
  8                -1  1    656896  models.common.SPP                       [512, 512, [5, 9, 13]]        
  9                -1  1   1182720  models.common.C3                        [512, 512, 1, False]          
 10                -1  1    131584  models.common.Conv                      [512, 256, 1, 1]              
 11                -1  1         0  torch.nn.modules.upsampling.Upsample    [None, 2, 'nearest']          
 12           [-1, 6]  1         0  models.common.Concat                    [1]                           
 13                -1  1    361984  models.common.C3                        [512, 256, 1, False]          
 14                -1  1     33024  models.common.Conv                      [256, 128, 1, 1]              
 15                -1  1         0  torch.nn.modules.upsampling.Upsample    [None, 2, 'nearest']          
 16           [-1, 4]  1         0  models.common.Concat                    [1]                           
 17                -1  1     90880  models.common.C3                        [256, 128, 1, False]          
 18                -1  1    147712  models.common.Conv                      [128, 128, 3, 2]              
 19          [-1, 14]  1         0  models.common.Concat                    [1]                           
 20                -1  1    296448  models.common.C3                        [256, 256, 1, False]          
 21                -1  1    590336  models.common.Conv                      [256, 256, 3, 2]              
 22          [-1, 10]  1         0  models.common.Concat                    [1]                           
 23                -1  1   1182720  models.common.C3                        [512, 512, 1, False]          
 24      [17, 20, 23]  1    229245  Detect                                  [80, [[10, 13, 16, 30, 33, 23], [30, 61, 62, 45, 59, 119], [116, 90, 156, 198, 373, 326]], [128, 256, 512]]
Model Summary: 283 layers, 7276605 parameters, 7276605 gradients

 time (ms)     GFLOPS     params  module
    180.91       0.00       3520  models.common.Focus
    153.11       0.00      18560  models.common.Conv
    321.00       0.00      18816  models.common.C3
     92.07       0.00      73984  models.common.Conv
    495.51       0.00     156928  models.common.C3
     74.00       0.00     295424  models.common.Conv
    403.86       0.00     625152  models.common.C3
     64.33       0.00    1180672  models.common.Conv
    162.37       0.00     656896  models.common.SPP
    183.09       0.00    1182720  models.common.C3
     37.09       0.00     131584  models.common.Conv
      5.72       0.00          0  torch.nn.modules.upsampling.Upsample
      9.12       0.00          0  models.common.Concat
    218.21       0.00     361984  models.common.C3
     28.29       0.00      33024  models.common.Conv
      6.74       0.00          0  torch.nn.modules.upsampling.Upsample
     12.23       0.00          0  models.common.Concat
    228.79       0.00      90880  models.common.C3
     50.36       0.00     147712  models.common.Conv
      5.35       0.00          0  models.common.Concat
    209.92       0.00     296448  models.common.C3
     43.30       0.00     590336  models.common.Conv
      0.35       0.00          0  models.common.Concat
    248.59       0.00    1182720  models.common.C3
     57.34       0.00     229245  Detect
3291.7ms total

**Comment by jaskiratsingh2000** [comment]
@glenn-jocher Can you give me the steps to produce this for yolov3-tiny?

**Comment by jaskiratsingh2000** [comment]
@glenn-jocher I have referenced this issue, so please let me know about it. I am still looking forward to your response on this. Thanks!
